distribution/sender/routes/request-consumer.xml

```diff
@@ -1,45 +0,0 @@
-<routes xmlns="http://camel.apache.org/schema/spring">
```

Comment:
Why did you delete this route?

Reply:
Do you remember the refactor that you've asked to be done in the first pull request?
There, we noticed that the **request-consumer** and **sender-message-consumer** would going to consume the same queue (openmrs.sync.{{db-sync.senderId}}).
So, you sugested to merge this two routes in one. Now, the logic that were in the **request-consumer** route was moved to **sender-message-consumer** route.
Thanks.

Reply:
See the message below from the first pull request:

_wluyima
 Collaborator
"I think we should use a single route to process all incoming activeMQ messages i.e. request-
consumer should be renamed to something else generic like sender-message-consumer
because it is a single queue we are reading from and then based on the message contents we
save the incoming message to the respective table i.e. sender_sync_request or
sync_response and then this route should be actually processing messages from the
sync_response table. It means we need to introduce a SyncResponse class mapped to a
sync_response table which this route will process."_

Reply:
I got it... I should have renamed the file **request-consumer** to **sender-message-consumer** instead, and then incorporate the new code from **sync-response-consumer**, in place of deleting **request-consumer** and use the **sync-response-consumer** file. This way, the git history would be more concise, with a renamed file rather than deleted.
My bad.